src/lib/tvgSceneImpl.h

```diff
@@ -119,8 +119,8 @@ struct Scene::Impl
 
         auto x1 = FLT_MAX;
         auto y1 = FLT_MAX;
-        auto x2 = 0.0f;
-        auto y2 = 0.0f;
+        auto x2 = -FLT_MAX;
```

Comment:
@JSUYA This solution is NOT platform independent, since the maximum negative value for floats depends on the float precision. If that's an issue I can use as the init values the values from the first paint:
```
        float x1, y1, x2, y2; 
        if (! (*paints.data)->pImpl->bounds(&x1, &y1, &x2, &y2)) return false;
        x2 += x1;
        y2 += y1;
```

Reply:
After talking with you, I did some research on `-FLT_MAX`.
-FLT_MAX seems to work well in standard(IEEE 754 as you say).
I think recalculating the first paint is unnecessary for now.
Therefore I think this change is appropriate.